exercises/practice/rotational-cipher/.meta/example.awk

```diff
@@ -0,0 +1,19 @@
+@load "ordchr"
+
+BEGIN {
+    FS = ""
+    OFS = ""
+    A = ord("A")
+    a = ord("a")
+}
+
+{
+    out = ""
+    for (i = 1; i <= NF; i++) {
+        if ("A" <= $i && $i <= "Z")
```

Comment:
Ah, my favourite way to write a "between" expression.

Reply:
No chained operators like Python supports :(